app/presenters/enabled_validations_presenter.rb

```diff
@@ -0,0 +1,32 @@
+class EnabledValidationsPresenter
+  def initialize(component)
+    @component = component
+  end
+
+  STRING_VALIDATIONS = %w[max_length min_length max_word min_word].freeze
+  STRING_LENGTH_POSTFIX = '_string_length'.freeze
```

Comment:
Maybe its a convention I'm unaware of, but I would have expected this to be `STRING_LENGTH_SUFFIX` as the opposite to a prefix...

Sorry, very petty I know.

Reply:
no problemo :) have updated the PR